config/config.go

```diff
@@ -158,15 +171,59 @@ const (
 	MetricsInflux MetricsType = "influx"
 )
 
-type Influx struct {
+type InfluxMetrics struct {
 	Host     string `mapstructure:"host"`
 	Database string `mapstructure:"database"`
 	Username string `mapstructure:"username"`
 	Password string `mapstructure:"password"`
+	Enabled  bool   `mapstructure:"enabled"`
+}
+
+func (influxMetricsConfig *InfluxMetrics) validateAndLogMetricsData() bool {
+	if influxMetricsConfig.Host == "" {
+		influxMetricsConfig.Enabled = false
```

Comment:
Instead of alerting the config directly, it might be better to return a boolean value and let the caller decide what to do with the info.

Reply:
makes sense, done